mlir/test/CAPI/mixr_cobj.cpp

```diff
@@ -202,11 +200,11 @@ static bool constructAndTraverseIr(MlirContext ctx) {
                        mlir::PassManager::Nesting::Implicit);
 
   mlir::migraphx::addHighLevelPipeline(pm);
-  pm.run(module);
+  (void)pm.run(module);
```

Comment:
why do we need this?

Reply:
`-Wunused-result` is the setting, IIRC. `LogicalResult` has a NODISCARD on it, so the compiler will yell if you don't check it or explicitly point out you're not checking it with something like that `(void)`

Reply:
Right, i'd have better consumed it. (void) should be fine for now.